pkg/kubelet/cadvisor/util.go

```diff
@@ -32,6 +35,17 @@ func CapacityFromMachineInfo(info *cadvisorapi.MachineInfo) v1.ResourceList {
 			int64(info.MemoryCapacity),
 			resource.BinarySI),
 	}
+
+	// if huge pages are enabled, we report them as a schedulable resource on the node
+	if utilfeature.DefaultFeatureGate.Enabled(features.HugePages) {
+		for _, hugepagesInfo := range info.HugePages {
```

Comment:
I thought we agreed to for this release, only support single huge page size, but I couldn't find where we ensure that.

Reply:
ah, good catch.  i wrote validation for pod to only consume one size, and forgot to add the validation for node to only report one size larger than 0.  will add now.